docs/topics/crud-mission-intro.adoc

```diff
@@ -2,7 +2,7 @@ include::crud-mission-intro-paragraph.adoc[]
 
 The Booster also demonstrates the ability of the HTTP application to locate and connect to a database in OpenShift. Each runtime determines in an opinionated manner to implement the connectivity solution that is best suited in the given case. The runtime can choose between using _JDBC_, _JPA_, or access _ORM_ APIs directly.
 
-The Booster application exposes an HTTP _CRUD_ API, which provides endpoints that allow you to manipulate data by performing  _CRUD_ operations over HTTP. The _CRUD_ operations are mapped to HTTP `Verbs`. The API uses JSON formatting to receive requests and return responses to the user. The user can also use an UI provided by the booster to use the application. Specifically, this Booster provides an application that allows you to:
+The Booster application exposes an HTTP Relational Database Backend API, which provides endpoints that allow you to manipulate data by performing  _CRUD_ operations over HTTP. The _CRUD_ operations are mapped to HTTP `Verbs`. The API uses JSON formatting to receive requests and return responses to the user. The user can also use an UI provided by the booster to use the application. Specifically, this Booster provides an application that allows you to:
```

Comment:
No. The application exposes a CRUD API, but it definitely doesn't expose a relational database API. Maybe just "exposes an HTTP API which provides ..."?